src/core/Akka.Persistence/AtLeastOnceDeliverySemantic.cs

```diff
@@ -463,7 +463,8 @@ public void SetDeliverySnapshot(AtLeastOnceDeliverySnapshot snapshot)
 
         public void Cancel()
         {
-            _redeliverScheduleCancelable.Cancel();
+            // need a null check here, in case actor is terminated before StartRedeliverTask() is called
+            _redeliverScheduleCancelable?.Cancel();
```

Comment:
As explained in the comment, this eliminated some `NullReferenceException`s I saw thrown in the `AtLeastOnceDeliveryActor` specs.